### PR TITLE
update Lawrencium page

### PR DESCRIPTION
**Pull request by rieanand**
## Description
Adding the Lawrencium page. 

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [X] Adding element number, symbol, and short description
 

## Status
- [X] Ready to go

### CODE REVIEW

File: elements/lawrencium.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Lawrencium
+---
+
+Symbol : Lr
+
+Atomic Number : 103cd
```
Comment (vincent-nguyen-sys) [inline elements/lawrencium.md:8]: Did you mean to leave this cd here? Other than that, this seems great!

Comment (rieanand) [reply elements/lawrencium.md]: Edited with new version! Thank you

File: elements/lawrencium.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Lawrencium
+---
+
+Symbol : Lr
+
+Atomic Number : 103
```
Comment (vincent-nguyen-sys) [inline elements/lawrencium.md:8]: This looks good now after the typo was removed.

**Review by vincent-nguyen-sys — APPROVED** [review]
This looks good now that the typo has been removed!

**Pull request merged**